spec/intl.html

```diff
@@ -1338,6 +1338,259 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinmonth" type="abstract operation">
+          <h1>
+            CalendarDateDaysInMonth (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns the number of days in the given month in the calendar represented by _calendar_.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinyear" type="abstract operation">
+          <h1>
+            CalendarDateDaysInYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthsinyear" type="abstract operation">
+          <h1>
+            CalendarDateMonthsInYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateinleapyear" type="abstract operation">
+          <h1>
+            CalendarDateInLeapYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns *true* if it falls within a leap year for the calendar represented by _calendar_ and *false* otherwise.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatefields" type="abstract operation">
+          <h1>
+            CalendarDateFields (
+              _calendar_: a String,
+              _fields_: a List of Strings,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a list of standard fields in _fields_ and returns a new list by adding relevant calendar-specific fields for the calendar represented by _calendar_.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemergefields" type="abstract operation">
+          <h1>
+            CalendarDateMergeFields (
+              _calendar_: a String,
+              _fields_: a List of Strings,
+              _additionalFields_: a List of Strings,
+            )
+          </h1>
```

Comment:
Should provide info about what this AO is supposed to do. Here's some info from the docs:

Here's the text from that other comment: 

> Custom calendars should override this method if they accept fields in `from()` or `with()` other than the standard set of built-in calendar fields: `year`, `month`, `monthCode`, and `day`. The input array contains the field names that are necessary for a particular operation (for example, `'monthCode'` and `'day'` for `Temporal.PlainMonthDay.prototype.with()`). The method should make a copy of the array and add additional fields as needed.
> 
> For example, a calendar what uses eras should check to see if the input contains the `year` field, and if it does, then return `[...inputFields, 'era', 'eraYear']`.